src/test/java/org/wso2/carbon/connector/integration/test/cerner/CernerConnectorIntegrationTest.java

```diff
@@ -0,0 +1,396 @@
+import org.wso2.connector.integration.test.base.ConnectorIntegrationTestBase;
+import org.wso2.connector.integration.test.base.RestResponse;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+/**
+ * Integration test
+ */
+public class CernerConnectorIntegrationTest extends ConnectorIntegrationTestBase {
+
+    private Map<String, String> eiRequestHeadersMap = new HashMap<String, String>();
+    private Map<String, String> apiRequestHeadersMap = new HashMap<String, String>();
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        String connectorName = System.getProperty("connector_name") + "-connector-" +
+                System.getProperty("connector_version") + ".zip";
+        init(connectorName);
+        eiRequestHeadersMap.put("Accept-Charset", "UTF-8");
+        eiRequestHeadersMap.put("Content-Type", "application/json+fhir");
+        eiRequestHeadersMap.put("Accept", "application/json+fhir");
+        apiRequestHeadersMap.putAll(eiRequestHeadersMap);
+    }
+
+    /**
+     * Positive test case for getConformance method with mandatory parameters.
+     */
+    @Test(enabled = true, description = "cerner {getConformance} integration test with mandatory parameters.")
+    public void getConformanceWithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:getConformance");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "getConformance.json");
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/metadata";
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+    }
+
+    /**
+     * Positive test case for readSpecificResourceById method with Mandatory parameters.
+     */
+    @Test(enabled = true, description = "cerner {readSpecificResourceById} integration test with Mandatory parameters.")
+    public void readSpecificResourceByIdWithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:retrieveById");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "readSpecificResourceById.json");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/" + connectorProperties.getProperty("type") + "/" + connectorProperties.getProperty("id");
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
```

Comment:
Shall we check at least one field from response.
Please refer: https://github.com/wso2-extensions/esb-connector-spotify/blob/master/src/test/java/org/wso2/carbon/connector/integration/test/spotify/SpotifyConnectorIntegrationTest.java#L154